source/content/global-cdn.md

```diff
@@ -32,6 +32,24 @@ We recommend installing the Pantheon Advanced Page Cache [plugin](https://wordpr
 
 For more details, see [Clearing Caches for Drupal and WordPress](/clear-caches).
 
+## Persistent Cache
+
+Serve your Drupal or WordPress site, even when it's down. If the server is not responding and can't serve a new copy of a page, it will use a cached version instead of displaying an error. With Persistent Cache, the goal is to provide a seamless, uninterrupted experience for the user.
+
+### How Do I Configure This Setting?
+
+On [Drupal](/drupal-cache#drupal-8-performance-configuration) and [WordPress](//wordpress-cache-plugin#pantheon-page-cache-plugin-configuration), you can adjust your CDN edge configuration to serve stale content for a specific amount of time.
```

Comment:
The settings built into the CMS don't impact how long the CDN will serve stale content, rather they impact how long the CDN is allowed to regard the content as "fresh." We've always supported this, whereas the new feature is not configurable at all by the customer.

Reply:
I combined two sections and renamed this one `Caching Exceptions - How long does content stay fresh?` for now - not sure I'm totally settled on it, but I think it's closer than giving the impression that the CDN is the configurable part of this feature